.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
I never found this to be a problem, I would leave this step as easy as possible and with a standard for `development` and `test` #RailsDeTodaLaVida

Then hackish people can override that option as they wish. In this way the development environment thing will be easier too.